codebase/src/java/disco/org/openlvc/disco/configuration/RprConfiguration.java

```diff
@@ -47,7 +54,8 @@
 	public enum RtiProvider
 	{
 		Portico("C:/Program Files/Portico", "lib/portico.jar" ),
-		Pitch("C:/Program Files/prti1516e", "lib/prtifull.jar" );
+		Pitch("C:/Program Files/prti1516e", "lib/prtifull.jar" ),
+		Mak("C:/MAK/makRti4.4", "lib/hla.jar" );
```

Comment:
Let's change the default path to just `C:/MAK`. It won't be correct, but pretty soon the v4.4 path won't be "correct" either (might already not be the latest). We used to have smarter code that tried to find the path, but it caused other issues. I'm pretty sure the Portico path has similar issues (includes the version in the path). So we might as well be all be wrong.